src/Model/ArrayData.php

```diff
@@ -87,6 +81,7 @@ public function getField(string $fieldName): mixed
     */
     public function setField(string $fieldName, mixed $value): static
     {
+        $this->objCacheClear();
```

Comment:
This matches `ModelData::setField()` and should have probably always been here. Clears the `obj()` cache if we're setting values.